script.js

```diff
@@ -0,0 +1,24 @@
+var GalaxyS6 = new Telefon('Samsung', 2200, 'czarny','', '10', '');
+var iPhone6S = new Telefon('Apple', 3300, 'srebrny','USD', '10', '');
+var OnePlusOne = new Telefon('OnePlus', 1100, 'bialy','EUR', '20', '');
+
+function Telefon(brand, price, color, currency, warrantyPercent, warrantyCost) {
```

Comment:
Ten `warrantyCost` w parametrze funkcji jest nie używany. Jeśli zrobiłaś go na "zaś" to ok, natomiast miej świadomość, że nie musi go tu być.